src/common/interfaces/IMessageDispatcher.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.5.0;
```

Comment:
Nit: 
```suggestion
pragma solidity 0.8.28;
```

Reply:
Interface pragmas should be `>=0.5.0`. This allows integrators of Centrifuge contracts to import the interfaces, regardless of the Solidity version they use.